src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -292,9 +303,9 @@ public String getPassword() {
         public String getRegistryUrl() { return registryUrl; }
```

Comment:
not doing anything with them for the credentials, but left them around, could be removed later